### DISCUSSION TITLE
"Create Row" button for Row Explorer block

### DISCUSSION THREAD

**Discussion by Grayda** [Budibase/budibase#10662] [Ideas]
I like the Table Block's "Create Row" button, because it makes adding new rows easy, and I like the Row Explorer block because it looks nice and makes searching, browsing (and editing) dead simple. I know I can manually add the sidebar with my required fields on there, but it'd be nice if the Row Explorer block had a built in "Create Row" button that either slid open the sidebar, or preferably, used the right-hand column to let you create the row.

**Comment by melohagan** [comment]
Hey @Grayda 

Using the right hand column makes sense to me! I've raised a feature request here: https://github.com/Budibase/budibase/issues/10698
